arch/src/x86_64/mod.rs

```diff
@@ -164,7 +170,14 @@ pub fn configure_system(
             configure_pvh(guest_mem, cmdline_addr, rsdp_addr)?;
         }
         BootProtocol::LinuxBoot => {
-            configure_64bit_boot(guest_mem, cmdline_addr, cmdline_size, setup_hdr, rsdp_addr)?;
+            configure_64bit_boot(
+                guest_mem,
+                cmdline_addr,
+                cmdline_size,
+                initramfs,
+                setup_hdr,
+                rsdp_addr,
+            )?;
```

Comment:
@rbradford I bet there's no restriction to boot from initramfs with the PVH protocol?
If that's the case, do we expect @gdamjan to implement initramfs for PVH in this PR too?

Reply:
seems to me it might need support in `hvm_start_info` first?